test/Spec.hs

```diff
@@ -0,0 +1 @@
+{-# OPTIONS_GHC -F -pgmF hspec-discover #-}
```

Comment:
Check out the [hspec docs](http://hspec.github.io/hspec-discover.html) for more info.